### PR TITLE
Feature/admin panel

### PR DESCRIPTION
**Pull request by Qouagga**
Add description to tables in Django administration

### CODE REVIEW

File: .gitignore
Comment (ixio) [inline .gitignore:81]: /opt shouldn't not be accessible from the git folder so are you sure this is needed ? Also be sure to always have a newline at the end of each file :wink: this is way here in github diff you have a red symbol

File: .vscode/settings.json
Comment (ixio) [inline .vscode/settings.json:2]: I'm not sure I want X IDE settings files for each new dev on the project :thinking:. Maybe you can add in the .gitignore and put it as a suggestion in the README.md ? Something like "If you are using VSCode you can add `"python.formatting.provider": "black"` to your `.vscode/settings.json` file :thinking:

File: backend/api/actions/datawork_import.py
Comment (ixio) [inline backend/api/actions/datawork_import.py:22]: Not sure why you removed this comment, it still feels relevant :sweat_smile:

File: frontend/package.json
Comment (ixio) [inline frontend/package.json:39]: Please don't change files un-related to the PR :wink:

File: frontend/src/AudioAnnotator/Workbench.js
Comment (ixio) [inline frontend/src/AudioAnnotator/Workbench.js:400]: Don't forget to remove logs :wink:

File: poetry.lock
Comment (ixio) [inline poetry.lock:7]: I don't believe you should change `poetry.lock` if there are no changes to `pyproject.toml` but this might be an error in my setup of poetry :thinking:. For now I'd rather you not merge these kind of changes :thinking: but we should double check that together

**Review by ixio — COMMENTED** [review]
I don't see any major issues, good job :) let's wait for our team feedback before merging though.

**Review by rvovard — APPROVED** [review]
Looks good, thanks @Qouagga for the good work :)

(And thank you @ixio for your review, I agreed with most of your comments!)

**Pull request merged**
